hushline/model.py

```diff
@@ -209,7 +219,7 @@ class SecondaryUsername(Model):
 class Message(Model):
     id: Mapped[int] = mapped_column(primary_key=True)
     _content: Mapped[Optional[str]] = mapped_column(
```

Comment:
Removing the `Optional` should indicate that this column is not nullable.
```suggestion
    _content: Mapped[str] = mapped_column(
```